### PR TITLE
fix(ProgressInidicator): remove null rendering

### PR DESCRIPTION
**Pull request by hellobrian**
fixes https://github.com/carbon-design-system/carbon-components-react/issues/201

### CODE REVIEW

File: components/ProgressIndicator.js [hunk 1]
```diff
@@ -100,20 +104,16 @@ class ProgressIndicator extends Component {
     });
 
   render() {
-    const { className, currentIndex, ...other } = this.props;
+    const { className, currentIndex, ...other } = this.props; // eslint-disable-line no-unused-vars
```
Comment (hellobrian) [inline components/ProgressIndicator.js]: Change: disable linting for unused `currentIndex` prop

File: components/ProgressIndicator.js [hunk 2]
```diff
@@ -100,20 +104,16 @@ class ProgressIndicator extends Component {
...
-      return (
-        <ul className={classes} {...other}>
-          {this.renderSteps()}
-        </ul>
-      );
-    }
-
-    return null;
+    return (
+      <ul className={classes} {...other}>
```
Comment (hellobrian) [inline components/ProgressIndicator.js]: Change: remove `null` rendering

**Comment by tw15egan** [comment]
👍 ✅

File: components/ProgressIndicator.js (hunk 1 above)
Comment (asudoh) [reply components/ProgressIndicator.js]: Another options is: https://eslint.org/docs/rules/no-unused-vars#ignorerestsiblings

**Comment by chrisdhanaraj** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/6b8b7163-1ec8-4616-82fa-aa467a17af03 "storybook-toolchain-v2"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/b84aaac8-8484-4a1e-902a-ebb526eb7998 "storybook-toolchain-v2") deployed [carbon-components-react](https://github.com/carbon-design-system/carbon-components-react/releases/tag/deploy-ibm-yp-us-south-chrisdhanaraj_org-Bluemix-Design-System-20171011-005532 "deploy-ibm-yp-us-south-chrisdhanaraj_org-Bluemix-Design-System-20171011-005532") to [Bluemix Design System](https://console.bluemix.net/apps/bc75a101-6472-481c-b48d-3eed1e9703f3?env_id=ibm:yp:us-south "ibm:yp:us-south:chrisdhanaraj_org:Bluemix Design System"), including commit 503bbfcd1cd7bb868ea947cc916452e0b82875e7

**Pull request merged**